easybuild/toolchains/mpi/mpich.py

```diff
@@ -46,19 +47,18 @@ class Mpich(Mpi):
 
     MPI_LIBRARY_NAME = 'mpich'
 
+    # FIXME version-dependent? see http://www.mpich.org/static/docs/v3.1.4/www1/mpifort.html
+    MPI_COMPILER_MPIFC = 'mpif90'
+    #MPI_COMPILER_MPIFC = 'mpifort'
```

Comment:
That's right. They suddenly switch the name in a **patch release**! This also makes the sanity check fail when using a v3.1.1 easyconfig with `--try-software-version`. With respect to MPICH2, there is no need to worry as the entire release series provided `mpif90`.

Note that the `mpifort` thing also applies to recent versions of Open MPI (see https://www.open-mpi.org/faq/?category=mpi-apps#mpifort-vs-mpif77-and-mpif90), though they automatically create symlinks to `mpif77` and `mpif90` for backwards compatibility.